qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/ExportingEndpointIT.java

```diff
@@ -0,0 +1,171 @@
+
+        try (final var client = cluster.newClientBuilder().build()) {
+          deployProcess(client);
+          startProcess(client);
+
+          final var recordsBeforePause =
+              Awaitility.await()
+                  .atMost(Duration.ofSeconds(30))
+                  .during(Duration.ofSeconds(5))
+                  .until(exportedRecords::size, hasStableValue());
+
+          // when
+          ExportingActuator.of(cluster.getAvailableGateway()).pause();
+          startProcess(client);
+
+          // then
+          Awaitility.await()
+              .atMost(Duration.ofSeconds(30))
+              .during(Duration.ofSeconds(10))
+              .failFast(() -> assertThat(exportedRecords).hasSize(recordsBeforePause));
+
+          Awaitility.await().untilAsserted(() -> allPartitionsPaused(cluster));
+        }
+      }
+    }
+  }
+
+  @Test
+  void failsIfMemberIsShutdown() {
+    try (final var cluster =
+        ZeebeCluster.builder()
+            .withImage(ZeebeTestContainerDefaults.defaultTestImage())
+            .withEmbeddedGateway(true)
+            .withBrokersCount(3)
+            .withPartitionsCount(1)
+            .withReplicationFactor(3)
+            .build()) {
+      // given
+      cluster.start();
+
+      // when
+      //noinspection resource
+      cluster.getBrokers().values().stream().findAny().orElseThrow().stop();
+
+      // then
+      final var actuator = ExportingActuator.of(cluster.getAvailableGateway());
+      assertThatExceptionOfType(FeignException.class)
+          .isThrownBy(actuator::pause)
+          .returns(500, FeignException::status);
+    }
+  }
+
+  @Test
+  void succeedsWithoutConfiguredExporters() {
+    try (final var cluster =
+        ZeebeCluster.builder()
+            .withImage(ZeebeTestContainerDefaults.defaultTestImage())
+            .withEmbeddedGateway(true)
+            .withBrokersCount(1)
+            .withPartitionsCount(1)
+            .withReplicationFactor(1)
+            .withNetwork(Network.newNetwork())
+            .build()) {
+      // given
+      cluster.start();
+
+      // then
+      assertThatCode(() -> ExportingActuator.of(cluster.getAvailableGateway()).pause())
+          .doesNotThrowAnyException();
+    }
+  }
+
+  private static void startProcess(final ZeebeClient client) {
+    client
+        .newCreateInstanceCommand()
+        .bpmnProcessId("processId")
+        .latestVersion()
+        .withResult()
+        .send()
+        .join();
+  }
+
+  private static void deployProcess(final ZeebeClient client) {
+    client
+        .newDeployResourceCommand()
+        .addProcessModel(
+            Bpmn.createExecutableProcess("processId").startEvent().endEvent().done(),
+            "process.bpmn")
+        .send()
+        .join();
+  }
+
+  private void allPartitionsPaused(final ZeebeCluster cluster) {
+    for (final var broker : cluster.getBrokers().values()) {
+      assertThat(PartitionsActuator.of(broker).query().values())
```

Comment:
:thought_balloon: This reminds me that partition actuator should also be moved from broker to the gateway/shared api :upside_down_face: